### PR TITLE
cosmetic(pie-button): new primary colour and larger size

### PR DESCRIPTION
**Pull request by jamieomaguire**
# This is not a real PR

[Changed] - Using a new colour for primary buttons and larger height for the large size

This PR introduces a new background colour for primary buttons, and increases the size to `70px` for `large` buttons.

### CODE REVIEW

**Comment by RLumleyJET** [comment]
This is a weird request

**Comment by nazanin** [comment]
Test

**Comment by jordifornellsJET** [comment]
😏

**Comment by tingting-lau** [comment]
Hi

File: packages/components/pie-button/src/button.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
 $button-height-xs: 32px;
 $button-height-s: 40px;
 $button-height-m: 48px;
-$button-height-l: 56px;
+$button-height-l: 70px;
```
Comment (jamieomaguire) [inline packages/components/pie-button/src/button.scss:6]: This seems wrong, has design requested this?

**Comment by jamieomaguire** [comment]
@RLumleyJET just checking you're happy with this change?

**Pull request closed without merging**